### DISCUSSION TITLE
Welcome to WindowsInput Discussions!

### DISCUSSION THREAD

**Discussion by TonyValenti** [MediatedCommunications/WindowsInput#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Valkirie** [comment]
Hey @TonyValenti, thank you very much for you project. I have an issue though.
I have the following Chord `new ChordClick(KeyCode.LWin, KeyCode.D)` that I want to fully intercept.

My listener successfully detects the input chord but I can't find a way to prevent the system from getting those two commands.

**Reply by TonyValenti** [reply]
I am actually not sure if it is possible to prevent the system from receiving the chord. It may be possible to ignore the last key press in the combination though.

**Reply by Valkirie** [reply]
Then maybe I can listen to key down, block input if it's the first key of one of my chords. Release after a few milliseconds if no other key is pressed or check if next keys are still part of a chord..

**Reply by TonyValenti** [reply]
You should likely only block if it is the last key. For example, if your chord is CONTROL+Q you should likely still allow the OS to receive CONTROL because they might click CONTROL+S.

**Reply by Valkirie** [reply]
Had the same idea but there are chords where both the first and second keys are "critical".
- IE: Windows key, Windows + D
